chart/kubeapps/requirements.lock

```diff
@@ -1,6 +1,6 @@
 dependencies:
 - name: mongodb
   repository: https://kubernetes-charts.storage.googleapis.com
-  version: 7.2.7
+  version: 7.2.9
 digest: sha256:415440e73af7d4b02a10a15f28bb2fc095cbdffdc2e1676d76e0f0eaa1632d50
```

Comment:
erm, checking why the digest didn't change...

Reply:
Apparently it doesn't change (the commit history shows it not changing) and I see now it's not a digest of the chart dependency.